### PR TITLE
docs: sphinx: Correct jai-imageio link

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Correct link and reference new upstream source repository.

### CODE REVIEW

File: ant/logback.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <pattern>%d [%t] %-5p %c - %m%n</pattern>
     </encoder>
   </appender>
-  <root level="WARN">
+  <root level="DEBUG">
```
Comment (mtbc) [inline ant/logback.xml]: Could you explain more about this change?

Comment (rleigh-codelibre) [reply ant/logback.xml]: An unintentional addition from a dirty tree; I've got rid of it.

**Comment by hflynn** [comment]
We don't in general use square brackets for docs, no, unless it's quoted code featuring them obviously

**Comment by sbesson** [comment]
Only action left here is to replace square brackets by curved ones, is that correct @hflynn ?

**Comment by hflynn** [comment]
Yes please.

**Comment by hflynn** [comment]
👍

**Comment by dgault** [comment]
Link is working and page displays correctly, PR is good to merge

**Pull request merged**